WordPressShared/WordPressShared/Core/Utility/NSString+Helpers.m

```diff
@@ -112,6 +112,14 @@ + (NSString *)stripShortcodesFromString:(NSString *)string
                                       withTemplate:@""];
 }
 
++ (NSString *)summaryFromContent:(NSString *)string
```

Comment:
doH indeed. 
I still think its too specific for an NSString category but I don't have a better idea atm. :/   Let's go with the change as is and if we think of something better later we can revisit.  Cool?